example/ck_tile/01_fmha/codegen/ops/fmha_fwd.py

```diff
@@ -410,7 +410,8 @@ def get_fmha_fwd_tile_dict_from_dtype(dtype : str) -> Optional[dict]:
     if dtype == 'fp16' or dtype == 'bf16':
         return {
             '32'  : FmhaFwdTileSize(128, 64, 16, 32, 32, 32,     2, 1, 1,  2, 1, 1,  32, 32, 16, -1),
-            '64'  : FmhaFwdTileSize(128, 64, 32, 64, 32, 64,     4, 1, 1,  4, 1, 1,  32, 32, 16, -1),
+            '64'  : FmhaFwdTileSize(64, 64, 32, 64, 32, 64,     4, 1, 1,  4, 1, 1,  16, 16, 16, -1),
```

Comment:
should add an if else to choose here.  And we had better use another PR for this change.